docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md

```diff
@@ -0,0 +1,302 @@
+---
+aliases:
+  - ../features/dashboard/dashboards/
+  - dashboard-manage/
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Externally shared dashboards
+menuTitle: Shared dashboards
+description: Make your Grafana dashboards externally shared and share them with anyone
+weight: 8
+refs:
+  dashboard-sharing:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/share-dashboards-panels/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/dashboards/share-dashboards-panels/
+  custom-branding:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/configure-custom-branding/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/configure-custom-branding/
+  dashboard-insights-documentation:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/dashboards/assess-dashboard-usage/#dashboard-insights
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/visualizations/dashboards/assess-dashboard-usage/
+  caching:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/#query-and-resource-caching
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/#query-and-resource-caching
+  grafana-enterprise:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana/<GRAFANA_VERSION>/introduction/grafana-enterprise/
+---
+
+# Externally shared dashboards
+
+<!-- Update aliases -->
+
+{{< admonition type="warning" >}}
+Sharing your dashboard externally could result in a large number of queries to the data sources used by your dashboard.
+This can be mitigated by using the Enterprise [caching](ref:caching) and/or rate limiting features.
+{{< /admonition >}}
+
+Externally shared dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to make your dashboard available to the world without requiring access to your Grafana organization.
+
+You can see a list of all your externally shared dashboards in one place by navigating to **Dashboards > Shared dashboards**. For each dashboard in the list, the page displays the status, a link to view the dashboard, a link to the externally shared dashboard configuration, and the options to pause or revoke the external URL.
```

Comment:
There are 2 dashboard links.
The first one redirects you to the real dashboard and is when you click the whole card
The second one (I think that's the one you refer to) ,is to view the public dashboard. Is it ok to say "a link to view the dashboard"? because that's not true, currently the public dashboard is not exactly as the dashboard (template variables, unsupported data sources, etc).
Should we say "a link to view the shared/external dashboard"?

Reply:
I'm addressing this on #90630